### PR TITLE
docs: Added explanation about period type @ programstage

### PR DESCRIPTION
**Pull request by dayures**
from https://community.dhis2.org/t/help-with-tracker/36123/12

### CODE REVIEW

File: src/user/configure-programs-in-the-maintenance-app.md [hunk 1]
```diff
@@ -1118,7 +1118,7 @@ by the data capture apps to identify this program stage.
 number of days to wait for starting the program stage.
 3. Enter repeatable program stage details.
     1. Specify if the program stage is **Repeatable** or not.
-    2. Select a **Period type**.
+    2. Select a **Period type** (when the period type on program stage works, it will mask the event date selection and replace it with a period picker).
```
Comment (Philip-Larsen-Donnelly) [inline src/user/configure-programs-in-the-maintenance-app.md:1121]: What is meant by "when the period type on program stage works"?

Comment (Philip-Larsen-Donnelly) [reply src/user/configure-programs-in-the-maintenance-app.md:1121]: @Bekkalizer perhaps you can clarify, as this is taken from your comment in the CoP :)

**Comment by larshelge** [comment]
Closing due to inactivity.

**Comment by dayures** [comment]
@Philip-Larsen-Donnelly could you follow up this issue, so it is not closed without any action ?

**Pull request closed without merging**